create_run.sh

```diff
@@ -25,14 +25,16 @@ read ram
 cat >run.sh <<EOF
 #!/bin/bash
 sysctl -w net.ipv4.ip_forward=1
-if [ -z "$1" ]
+if [ -z "\$1" ]
```

Comment:
Was macht das?

Reply:
"$1" wird von create_run.sh als variable genommen und somit der wert ("") in die run.sh geschrieben.
Mit dem Backslash wird auch wirklich der string "$1" in run.sh geschrieben.